index.html

```diff
@@ -110,7 +110,37 @@ <h1>Introduction</h1>
 <section>
 <h1>Comments</h1>
 <p>
-Lorem ipsum....
+<b>For Document 800-63A:
+Section 8.1.1 Social Security Numbers</b>
+  <p>
+
+"...Overreliance on the SSN can contribute to misuse and place the applicant at risk of harm, such as through identity theft. Nonetheless, ...."
+  </p>
+  <b>
+W3C CCG Group Comments:
+  </b> 
+  <p>
+For some use cases, when Privacy concern is high, one time use of DID identifier or the hash of the identifier can be used as a unique alternative identifier to the SSN. For further privacy, the pairwise DID identifier can be used. The DID identifier is meaningless by itself, but globally unique;it does not leak PII or any sensitive data. Public exposure of the DID identifier does not allow for it to be used as an authenticator or shared secret without a digital signature and DID Auth process. Verifiable Confidential and DID document can be constructed using context property to allow use the same DID identifier within same context. For example, for any government service, under user’s consent, the same DID identifier can be used across systems, agencies and organizations without compromising its security and privacy properties.
+
```

Comment:
Replace 
The DID identifier is meaningless by itself, but globally unique;it does not leak PII or any sensitive data
with
Whilst the DID identifier is a globally unique identifier, and thus is PII, nevertheless it is meaningless by itself. It does not leak any additional PII or sensitive data.

Reply:
Agree with @https://github.com/David-Chadwick commens